testing/Overview.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import {mount, shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ItemDescription from '../client/src/Components/itemDescription.jsx';
+import ShippingInformation from '../client/src/Components/ShippingInformation.jsx'
+import globalFetch from 'jest-fetch-mock';
+import sinon from 'sinon'
+import Overview from '../client/src/Components/Overview.jsx'
+import Cart from '../client/src/AddToCart.jsx'
+import ReactDOM from 'react-dom'
+import ReturnPolicy from '../client/src/Components/ReturnPolicy.jsx';
+import SelectionList from '../client/src/Components/SelectionList.jsx';
+import renderer from 'react-test-renderer';
+
```

Comment:
It looks like you need to import `returnPolicy` and `ItemDescription`, I didnt see the other modules being used in this file.  You only need to import modules you directly call; if a module needs to mount a submodule, it will grab it from the file itself.